packages/react-button/src/components/Button/Button.stories.tsx

```diff
@@ -1,168 +1,42 @@
-import * as React from 'react';
-import { Button, ButtonProps } from '../../Button';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore - add link to this comment please  (see https://github.com/microsoft/fluentui/pull/18695)
```

Comment:
This will break any type checking related to the component, which is probably not a good idea for maintainability or our internal dev experience. What was the reason you had to it this way? Then maybe we can come up with an alternative.

Reply:
As a general principle, we should optimize for the primary scenario (local dev experience and ensuring types stay in sync), and if that doesn't work with some secondary scenario (export to codesandbox), any workarounds/hacks should be done in the secondary scenario's code.

Reply:
The main reason for this is that we build storybook code along with other code. During the `build` process, we no longer use ts path aliases, which means that without this ts-ignore, the build will always fail. We want to use aliases here because we want to avoid circular dependencies between packages.

As long as we use ts path aliases, the only long term solution for this is to use separate ts configs which is proposed in #19044

Reply:
Would it be possible to make the codesandbox export addon convert relative imports within the same package to absolute imports? That would at least get rid of some of the ts-ignores.

Especially if we end up renaming slots to be named with a `Slot` suffix, having type checking disabled all over the place is a huge risk.

Reply:
Is there any way the addon could access package.json to read the package name and version? Or could they be read and passed in when configuring the addon?

Reply:
I think that would require more time than I can commit to this. I can create a GitHub issue about it if you think it’s worth it?

Reply:
It's definitely worth doing *something* to avoid using a sledgehammer that so significantly detracts from maintainability. I did something sort of similar with `packages/react-examples/.storybook/preview-loader.js`, which is solving a different problem but the approach might be generally applicable. I'll check out your branch and see if I can come up with anything.

Reply:
@ecraig12345 I’ll merge this now without changing Button stories. Let’s address automatic version detection / absolute imports in other PRs.